addons/resource.language.en_gb/resources/strings.po

```diff
@@ -38,7 +45,7 @@ msgstr ""
 
 #: Used in Confluence
 msgctxt "#4"
-msgid "TV-Guide"
+msgid "TV-guide"
```

Comment:
maybe other skins still use this string even though Confluence doesn't any more. This is the reason we need to be careful with removing strings in the future.
@uNiversaI ^^ regarding your cleanup intentions. Let's discuss that on the forum on the proper steps to take.